__tests__/app.test.js

```diff
@@ -174,14 +191,29 @@ describe("GET /api/users", () => {
         expect(body.msg).toBe("Not Found");
       });
   });
-  test("404: Invalid endpoint inputted AND if no users in db", () => {
+
+  test("404: Invalid endpoint inputted", () => {
     return request(app)
       .get("/api/userss")
       .expect(404)
       .then(({ body }) => {
         expect(body.msg).toBe("Not Found");
       });
   });
+
+  test("404: Invalid if no users in db", () => {
```

Comment:
Make sure you're focusing on what the ticket is asking of you. In a development team, you'll be accountable for your 'velocity', which is a measure of how quickly you add functionality. If you're adding stuff that's not been asked of from the client, you're taking dev time from the stuff the client is paying you to do. 

Also users will always be there, and truncating tables is a risky command to be using in any circumstance with users.